src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,255 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { ALERT_SIZE, ALERT_TYPE, ALERT_ICON } from "./enums";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: "alert",
+          intro:
+            "Use the alert (or notice) component to keep the user updated about important information.",
+        }),
+      },
+    },
+  },
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+    buttonText: "Fout melden",
+    slotTitle: "Alert titel via slot",
+    content: "U heeft geen rechten om deze actie uit te voeren.",
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
+      type: { summary: "string" },
+      description: "Attribuut wordt gebruikt om de titel te bepalen.",
+      table: {
+        defaultValue: { summary: "" },
+      },
+    },
+    icon: {
+      name: "data-vl-icon",
+      type: {
+        summary: `${ALERT_ICON.WARNING} | ${ALERT_ICON.CHECK} | ${ALERT_ICON.INFO_CIRCLE}`,
+      },
+      description: "Attribuut wordt gebruikt om het icoon type te bepalen.",
+      control: {
+        type: "select",
+        options: [ALERT_ICON.WARNING, ALERT_ICON.CHECK, ALERT_ICON.INFO_CIRCLE],
+      },
+      table: {
+        defaultValue: { summary: `"${ALERT_ICON.WARNING}"` },
+      },
+    },
+    size: {
+      name: "data-vl-size",
+      type: {
+        summary: `${ALERT_SIZE.NORMAL} | ${ALERT_SIZE.SMALL}`,
+      },
+      description:
+        "Attribuut activeert een variant van de waarschuwing maar kleiner.",
+      control: {
+        type: "select",
+        options: [ALERT_SIZE.NORMAL, ALERT_SIZE.SMALL],
+      },
+      table: {
+        defaultValue: { summary: `"${ALERT_SIZE.NORMAL}"` },
+      },
+    },
+    type: {
+      name: "data-vl-type",
+      type: {
+        summary: `${ALERT_TYPE.INFO} | ${ALERT_TYPE.SUCCESS} | ${ALERT_TYPE.WARNING} | ${ALERT_TYPE.ERROR}`,
```

Comment:
Ben je zeker dat enkel die iconen kunnen? Of kan je gewoon kiezen wel icoon je gebruikt?

Reply:
Deze zaten in de demo pages. Je kan kiezen welk icoon je gebruikt voor zover ik weet.